src/sonic-config-engine/smartswitch_config.py

```diff
@@ -14,25 +15,29 @@
 except KeyError:
     pass
 
-DPUS_TABLE  = 'DPUS'
+DPU_TABLE = 'DPU'
```

Comment:
Why do we have these two?

Reply:
`DPUS` indicates that the image is running on the switch.
`DPU` indicates that the image is running on the DPU.
https://github.com/sonic-net/SONiC/blob/master/doc/smart-switch/ip-address-assigment/smart-switch-ip-address-assignment.md#device-data-and-platform

Reply:
Based on the type of device we can generate different configurations for switch and DPU for the same topology